docs/gettingstarted.rst

```diff
@@ -54,3 +54,4 @@ see example notebooks:
 * `Example Gaussian Synchrotron <https://gist.github.com/zonca/51a6fa9763106c78813f964a4b88f0fc>`_
 * `Example Gaussian Dust <https://gist.github.com/zonca/4ddb5e384cb34f8a2945c041d13e9428>`_
 * `Example InterpolatingComponent <https://gist.github.com/zonca/08751497b040ec9d62ff5175573c786e>`_
+* `Example Websky Extragalactic <https://gist.github.com/marcelo-alvarez/b13afc4a761c61334d11b5eeae953923>`_
```

Comment:
thanks for providing an example notebook!

Reply:
hope it helps